source/unified-test-format/schema-1.3.json

```diff
@@ -263,6 +348,7 @@
         "name": { "type": "string" },
         "object": { "type": "string" },
         "arguments": { "type": "object" },
+        "ignoreResultAndError": { "type": "boolean" },
```

Comment:
Since this is meant to be mutually exclusive with `expectError` and `expectResult`, I would suggest adding additional rules to the `allOf` definition for `operation`. See: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/schema-1.2.json#L270

IMO, this would be a fruitful learning exercise in writing the JSON schema, since everyone is likely to make changes to the test format down the line. This would also allow you to add tests to the test runner itself to ensure your schema changes are correct.

See https://github.com/mongodb/specifications/commit/62c6bb9ed030e91cdab8622299bb1bf970b7f68d#diff-25ec19412b8ee6ccb0d93ebcd467ba7958e722164db42c5fbcc9e38de3ae2fd8 for prior art.

Reply:
Added three new clauses.